lib/Doctrine/ORM/ORMException.php

```diff
@@ -324,16 +324,6 @@ public static function unrecognizedIdentifierFields($className, $fieldNames)
     }
 
     /**
-     * @param string $functionName
-     *
-     * @return ORMException
-     */
-    public static function overwriteInternalDQLFunctionNotAllowed($functionName)
```

Comment:
This method removal is effectively a BC break, and while it is an internal method, it should be documented in `UPGRADE.md`